x/sysproxy/README.md

```diff
@@ -0,0 +1,32 @@
+# Platform support
+
+The table below summarizes system-wide support this package offers for various types of proxies on desktop platforms.
+
+| Proxy Type | Windows | MacOS | Linux |
+| --- | ----------- | ------ | ------ |
+| HTTP | Yes | Yes | Yes
+| HTTPS | Yes | Yes | Yes
+| SOCKS | No | Yes | Yes
+
+
+`SetupWebProxy` implementation in this package setups both HTTP and HTTPS proxy when they distinguished by the platform. `SetupSOCKSProxy` sets up SOCKS proxy that proxies all TCP-based connections.
+
+For example, on MacOS, if you have HTTP Proxy setup and visit a website over HTTPS, your traffic does NOT go through the proxy. 
```

Comment:
Can you rearrange the notes. Perhaps in a "Platform considerations" section. Then group each OS and pull the "other notes".

Reply:
@fortuna I rewrote the godoc and rearranged the notes.